docs/web_reference.rst

```diff
@@ -2798,6 +2798,8 @@ Utilities
                                    closed after a HTTP request. The delay
                                    allows for reuse of a TCP connection.
 
+      .. versionadded:: 3.7
```

Comment:
```suggestion
      .. versionadded:: 3.8
```
Sorry, the time window for 7.1 is closed.

Reply:
Understood.